gui/resources.xml

```diff
@@ -1650,6 +1652,7 @@ Vocabulary
           <property name="icon-name">mypaint-tool-inking-symbolic</property>
           <signal name="activate" handler="mode_flip_action_activated_cb"/>
         </object>
+        <accelerator key="y"/>
```

Comment:
If I add a modifier to the move layer tool. We could assign the M key to the inking tool.